core/pom.xml

```diff
@@ -64,6 +64,19 @@
             <scope>test</scope>
         </dependency>
 
+        <!--TODO: Clean up version numbers-->
+        <dependency>
+            <groupId>uk.co.datumedge</groupId>
+            <artifactId>hamcrest-json</artifactId>
+            <version>0.2</version>
```

Comment:
Missing `<scope>test</scope>`?

Reply:
Yup. also need to move the version number to the parent pom.xml